Mod Menu Helper/assets/modmenu/lang/tr_TR.json

```diff
@@ -0,0 +1,55 @@
+  "modmenu.summaryTranslation.farsight": "§c*§r Sunuculardaki yüklü yığınları yüklü tutar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.fastchest": "§c*§r Daha hızlı sandıklar\n§9✎§8 Modu aç/kapa",
+  "modmenu.summaryTranslation.fastopenlinksandfolders": "Bağlantı veya klasörleri açarken donmayı önler\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.ferritecore": "RAM kullanımı azaltır\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.forgetmechunk": "DFixes lag spikes on chunk borders\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.hydrogen": "Reduce RAM usage\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.iris": "Optifine'ın shaderları\n§9✎§8 Modu aç/kapa, shader, tuş atamaları vb. seç",
+  "modmenu.summaryTranslation.itemmodelfix": "Elde tutulan eşyadaki görünmüz çizgileri düzeltir\n§9✎§8 Metod",
+  "modmenu.summaryTranslation.lambdabettergrass": "Optifine'daki \"daha iyi çimenler\"\n§9✎§8 Kalite, kar, kaynak paketi",
+  "modmenu.summaryTranslation.lambdynlights": "Optifine'ın \"dinamik ışıklandırmaları\"\n§9✎§8 Kalite, varlık seçenekleri, ...",
+  "modmenu.summaryTranslation.languagereload": "Hızlı dil değiştirme, arama çubuğu\n§9✎§8 Dili yeniden yüklemek için F3+J",
+  "modmenu.summaryTranslation.lazydfu": "Daha hızlı oyun açılışı\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.lithium": "Fizikleri ve yığınları optimize eder\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.modmenu": "Bu mod menüsü\n§9✎§8 Tuş yerleşimi, filtreleme, sıralama, ...",
+  "modmenu.summaryTranslation.morechathistory": "Sohbet geçmiş limitini arttırır\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.nofade": "Yüklenirken solma animasyonunu kaldırır\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.notenoughcrashes": "Çökmeden sonra bildir ve oyna\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.phosphor": "§c*§r Işığı optimize eder, daha hızlı yığınlar\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.puzzle": "Bazı Optifine özellikleri\n§9✎§8 Ayarlamak için modu genişlet( ▼ simgesine tıkla)",
+  "modmenu.summaryTranslation.puzzle-base": "Bazı Optifine özellikleri\n§9✎§8 Kaynak paketinden açılış ekranı, ...",
+  "modmenu.summaryTranslation.smoothboot": "Oyun donmalarını azaltır\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.sodium": "Görüntülemeyi optimize eder, daha iyi FPS\n§9✎§8 Ekran Ayarları'ndan: optimizasyonlar",
+  "modmenu.summaryTranslation.starlight": "Işığı optimize eder, daha hızlı yığınlar\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.reeses-sodium-options": "Dikey ekran ayarları\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.sodium-extra": "Optifine'ın grafik seçenekleri\n§9✎§8 Ekran Ayarlar'ından: animasyonlar, parçacıklar, ...",
+  "modmenu.summaryTranslation.tiefix": "Tanılayıcı veri gönderimini devre dışı bırakır, hataları düzeltir\n§9✎§8 Tanıları ve hata çözümlerini aç/kapa",
```

Comment:
Second line is not displayed, moves to a new row after "bırakır,"

Reply:
Tried to narrow it down. 👍 You may want to check TieFix mod desc.